Default colors

ejelly opened the pull request:
To close issue #24, I implemented the ability to configure a default background and foreground color. If at least one of those is set, the console output will be contained within a div with the appropriate style attribute set. Note that #24 called for setting this in the color map, but I opted for a per-job configuration instead.

ejelly commented:
By the way, I did not update the README since I think the configuration is self-explanatory.

dblock commented on src/main/java/hudson/plugins/ansicolor/AnsiColorBuildWrapper.java:
```diff
@@ -86,31 +99,35 @@ public Environment setUp(AbstractBuild build, Launcher launcher,
-                super.close();
-            }
-        };
+		if (logger != null)
```
Make this a bit more readable with some `{}` or maybe even with `if (logger == null) { return ... }`

dblock commented on src/main/java/hudson/plugins/ansicolor/AnsiColorBuildWrapper.java:
```diff
@@ -86,31 +99,35 @@ public Environment setUp(AbstractBuild build, Launcher launcher,
+					super.close();
+				}
+			};
+		else
```
```
else {
  return null;
}
```

dblock commented:
Looks good, can you fix the cosmetics and squash your commits please? I'll merge.

ejelly commented:
I changed the default fg/bg attributes to be part of the colormap, changed the "None" text do "Jenkins Default" (to make it clearer that you don't have to select something if you want ansicolor to leave the default foreground and background alone) and squashed the commits.

dblock commented:
Very nice, merging.

dblock commented:
You might want to update README maybe with a newer configuration screenshot, maybe with some notes on fg/bg colors?

ejelly commented:
Okay, I'll do that.
